### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by jiahuilu97** [muralits98/Predicting-Wild-Fire-with-Weather-Data-in-the-US-Geography#12]
In the project, the team predicted fires in many states given weather data. Then they found that the output accuracy depended upon the methods they picked. The size of the dataset in the Fires table is: 18, 800, 465×39, the size of the weather table is: 1, 629, 108×10 and the size of the final table is: 1, 629, 108 × 11. The team then used XGBoost, SVM, Decision Trees, DBScan, Boosting and Balanced Accuracy to build the models. 

The things I love:
1. The visualization is really cool. The website for row data shows the dynamic graphs of fire clearly and the graphs in the report makes the result easy to understand

2. Used some advanced techniques such as DBScan.

3. The project is meaningful and interesting. This can help us know what kind of weather should we pay extra attention to for preventing the fires.

Things I think can be improved:

1. We shouldn’t use the techniques as black box. The project used many techniques other than the ones mentioned in class. That’s cool, but I think you should give us more information about the idea behind, like the equations of each algorithms. The main point of this course is both to know the algorithm and to apply it in practice, right?

2. Maybe you can talk about the weapon of math in the report.

3. Maybe you can use more thing in the class, like loss function and so on.
